aiida_quantumespresso/calculations/__init__.py

```diff
@@ -872,8 +872,11 @@ def create_restart(self, force_restart=False, parent_folder_symlink=None,
             # add the restart flag
             old_inp_dict['CONTROL']['restart_mode'] = 'restart'
             inp_dict = ParameterData(dict=old_inp_dict)
-
-        remote_folders = self.get_outputs(node_type=RemoteData)
+        # Maintain compatibility with aiida_core 0.12.0 and older versions
+        try:
+            remote_folders = self.get_outputs(type=RemoteData)
```

Comment:
Please remove this try/catch. This version will only support 0.12.* and older anyway. In the `v3*` series we will make sure to update the signature for `aiida-core v1.0.0` and higher.

Reply:
Sorry, I get confused when this change of method signature was made - it was there since `0.11.2`. The `try`  was for maintain compatibility with versions older than that and not related to `v1.0.0` and higher. 
Here is the commit that made the change https://github.com/aiidateam/aiida_core/commit/991596e911f7c91b847aa0293d39f885610757b5.

Reply:
I have remove this commit.